datameta/api/files.py

```diff
@@ -25,106 +25,113 @@
 from pyramid.request import Request
 from pyramid.httpexceptions import HTTPOk
 from typing import Optional, Dict
-from .. import models
-
+from datetime import datetime, timedelta
+from .. import models, siteid, security, storage
+from . import DataHolderBase
 
 # For Developers: if needed change the dataclasses to
 # regular classes and supply custom __init__ functions
 # (for instance if create of that class object should
 # also trigger creation of the respective db model)
```

Comment:
This can be removed. Was just a message from me to you as the final developer.

Reply:
OK, I removed it